lisa/operating_system.py

```diff
@@ -406,6 +399,20 @@ def _install_package_from_url(
         pkg = wget_tool.get(package, str(self._node.working_path))
         self.install_packages(pkg, signed)
 
+    def wait_running_process(self, process_name: str, timeout: int = 5) -> None:
```

Comment:
With my latest change, it may not be able to merge. It may need a pidof tool to wait.